src/main/java/org/sagebionetworks/web/client/widget/entity/WikiPageWidgetViewImpl.java

```diff
@@ -145,7 +222,7 @@ private void showDefaultViewWithWiki() {
 			FlowPanel noticePanel = createDifferentVersionNotice();
 			add(wrapWidget(noticePanel, "alert alert-"+BootstrapAlertType.WARNING.toString().toLowerCase()+" wikiVersionNotice"));
 		}

```

Comment:
couldn't hide / remove noticePanel. I have tried a to set noticePanel and noticeWidget as global variables but it causes some bugs. Maybe I'll try again later. Do you have any suggestion on how to handle this?